zap/src/main/java/org/zaproxy/zap/utils/HirshbergMatcher.java

```diff
@@ -20,12 +20,12 @@
 package org.zaproxy.zap.utils;
 
 /**
- * This class provides the implementation of Hirschberg's algorithm that solves the Longest Common
```

Comment:
This is the correct spelling.

Reply:
Please note that the filename and the majority of the corpus get this wrong.

I made a guess based on apparent consensus. I'll switch.